{{cookiecutter.doc}}/index.rst

```diff
@@ -66,7 +66,7 @@
 Overview
 ========
 
-:doc:`Contact info <ts_xml:index:master-csc-table:{{cookiecutter.csc_name}}>`
+:ref:`Contact info <ts_xml:index:master-csc-table:{{cookiecutter.csc_name}}>`
```

Comment:
same comment as above about how to fix this... which admittedly should have been here instead of there....